Userland/Libraries/LibJS/Runtime/DatePrototype.cpp

```diff
@@ -271,6 +396,12 @@ JS_DEFINE_NATIVE_FUNCTION(DatePrototype::to_iso_string)
     auto* this_object = typed_this(vm, global_object);
     if (!this_object)
         return {};
+
+    if (this_object->is_invalid()) {
+        vm.throw_exception<RangeError>(global_object, ErrorType::InvalidTimeValue, "Date.toISOString (<anonymous>)");
```

Comment:
The function name should *not* be part of the error message, you've misinterpreted output from NodeJS's stack trace here :smile: